pkg/controller/dbaasregistration/configmap.go

```diff
@@ -0,0 +1,101 @@
+    mediatype: image/png
+  inventory_kind: MongoDBAtlasInventory
+  connection_kind: MongoDBAtlasConnection
+  credentials_fields: |
+    - key: orgId
+      display_name: Organization ID
+      type: string
+      required: true
+    - key: publicApiKey
+      display_name: Public API Key
+      type: string
+      required: true
+    - key: privateApiKey
+      display_name: Private API Key
+      type: maskedstring
+      required: true`
+)
+
+// +kubebuilder:rbac:groups="",resources=namespaces,verbs=get;list
+// +kubebuilder:rbac:groups="",resources=configmaps,resourceNames=mongodb-atlas-registration,verbs=get;list;create
+// +kubebuilder:rbac:groups="",namespace=default,resources=namespaces,verbs=get;list
+// +kubebuilder:rbac:groups="",namespace=default,resources=configmaps,resourceNames=mongodb-atlas-registration,verbs=get;list;create
```

Comment:
just try to understand why namespace=default?

Reply:
These two rows are used to generate a namespace level role which is used when the operator is deployed at the namespace level. A default namespace is used for such a purpose. When the operator is deployed in a target namespace at the namespace level, the actual namespace will be used by OLM.